src/Form/ColorChoiceType.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace AppBundle\Form;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ColorChoiceType extends AbstractType
+{
+    public function configureOptions(OptionsResolver $resolver): void
+    {
+        $resolver->setDefaults([
+            'choice_label' => function ($value) {
+                return $value;
+            },
+            'expanded' => true,
+            'choice_attr' => function () {
+                return [
+                    'style' => 'display:none;',
+                ];
+            },
+        ]);
+    }
+
+    public function getParent(): string
+    {
+        return ChoiceType::class;
+    }
+
+    public function finishView(FormView $view, FormInterface $form, array $options): void
+    {
+        foreach ($view->children as $child) {
```

Comment:
Could you explain in a comment what's the goal here please ? It would avoid us to look for twig files to understand.